pennylane/drawer/draw.py

```diff
@@ -399,6 +355,7 @@ def draw_mpl(
     style=None,
     *,
     fig=None,
+    level: Optional[Union[Literal["top", "user", "device", "gradient"], int, slice]] = "gradient",

```

Comment:
I _think_ this is correct since the default is not `None`.

```suggestion
    level: Union[None, Literal["top", "user", "device", "gradient"], int, slice] = "gradient",

```